source/fundamentals/run-command.txt

```diff
@@ -0,0 +1,165 @@
+.. _node-run-command:
+
+=============
+Run a Command
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to run a database command with the
+{+driver-short+}. You can use database commands to perform a variety of
+administrative and diagnostic tasks, such as fetching server statistics,
+initializing a replica set, or running an aggregation pipeline.
+
+Execute a Command
+-----------------
+
+To run a database command, you must specify the command and any relevant
+parameters in a command document, then pass the command document to a
+command execution method. The {+driver-short+} provides the following methods
+to run database commands:
+
+- ``command()``, which returns the command response as a
+  ``Document`` type. You can use this method with any database command.
+- ``runCursorCommand()``, which returns the command response as an iterable
+  ``RunCommandCursor`` type. You can use this method if your database command
+  returns multiple result documents.
+
+The following code shows how you can use the ``command()``
+method to run the ``hello`` command, which returns information about
+the current member's role in the replica set, on a database:
+
+.. code-block:: javascript
+
+   const result = await myDB.command({ hello: 1 });
+
+For a full list of database commands and corresponding parameters, see
+the :ref:`Additional Information section <addl-info-runcommand>`.
+
+.. note:: Read Preference
+
+   ``command()`` and ``runCursorCommand()`` do not obey the read
+   preference you may have set on your ``Db`` object elsewhere in
+   your code. You can set a read preference for command execution by
+   passing the relevant options object to either method:
```

Comment:
Q: What makes the objects "relevant"? Does each method accept a different object format?

Reply:
Yes, RunCommandOptions and RunCursorCommandOptions. I can clarify the pairs